src/pages/gas.tsx

```diff
@@ -0,0 +1,474 @@
+        "Gas keeps Ethereum sybil-resistant by preventing malicious actors from overwhelming the network with fraudulent activities.",
+    },
+    {
+      emoji: ":money_with_wings:",
+      description:
+        "Because computation costs gas, spamming Ethereum with expensive transactions, either accidentally and maliciously, is financially disencentivised.",
+    },
+    {
+      emoji: ":hourglass_flowing_sand:",
+      description:
+        "A hard-limit on the amount of computation that can be done at any one time prevents Ethereum from being overwhelmed, helping to ensure the network is always accessible.",
+    },
+  ]
+
+  const heroContent = {
+    title: "Network fees",
+    header: "Network fees",
+    subtitle:
+      "Network fees on Ethereum are called gas. Gas is the fuel that powers Ethereum.",
+    image: getImage(data.infrastructure)!,
+    alt: "",
+    buttons: [
+      {
+        content: "What is gas?",
+        toId: "explore",
+        matomo: {
+          eventCategory: "gas hero buttons",
+          eventAction: "click",
+          eventName: "wgat is gas",
+        },
+      },
+    ],
+  }
+
+  return (
+    <Page>
+      <PageMetadata
+        title="Gas fees on Ethereum: how do they work?"
+        description="Learn about gas on Ethereum: how they work and how to pay less in gas fees"
+      />
+      {/* Hero Section */}
+      <Box background="layer2Gradient" width="full">
+        <Box pb={8}>
+          <PageHero content={heroContent} isReverse />
+        </Box>
+      </Box>
+      <>
+        {/* Introduction / What are gas fees */}
+        <Content mb={{ base: 16, md: 16, lg: 32 }} mt={16}>
+          <Flex
+            direction={{ base: "column", lg: "row" }}
+            align="flex-start"
+            w="full"
+          >
+            <Box flex="60%" w="full" mr={{ base: "auto", lg: 2 }}>
+              <Box mb={16}>
+                <Text>
+                  Every transaction on Ethereum requires a small form of payment
+                  to process—these fees are known as ‘gas’ fee. Just like you
+                  need to pay for postage to send a letter, Ethereum requires
+                  you to pay gas fee to send a transaction.
+                </Text>
+              </Box>
+              <H2 mt={0}>What are gas fees?</H2>
+              <Text>
+                Think of Ethereum as a large computer network where people can
+                do tasks like sending messages or running programs. Just like in
+                the real world, these tasks require energy to get done.
+              </Text>
+              <Text>
+                In Ethereum, each computational action has a set "gas" price.
+                Your gas fees are the total cost of the actions in your
+                transaction. When you send a transaction or run a smart
+                contract, you pay in gas fees to process it.
+              </Text>
+            </Box>
+
+            <Show above="lg">
```

Comment:
Causing layout shift

Reply:
this is the example of layout shifting
https://github.com/ethereum/ethereum-org-website/assets/1120748/82b2e275-5fb9-4157-8dcf-3e2c3d7ecae2

Reply:
Fixed [in this commit](https://github.com/ethereum/ethereum-org-website/pull/11004/commits/f73b547d28486a108b972977ff0a14a92a82812d)

We saw a momentary flash of unstyled content using Chakra, causing [quite a lot of layout shift on this page](https://github.com/ethereum/ethereum-org-website/assets/1120748/82b2e275-5fb9-4157-8dcf-3e2c3d7ecae2). 

I tried reversing it from `Show above="lg"` (causing flash of unstyled on larger screens) to `Hide below="lg"` (mostly hidden layout shift on mobile), but that didn't seem ideal either:

![d121cac575e4073924a080aa9503158a](https://github.com/ethereum/ethereum-org-website/assets/62268199/ca350f0b-7958-41c5-b0c1-2c6576c988b2)

To fix the issue, I've used display none/display flex based on breakpoint which seems to be working for now.

cc: @pettinarip